src/actions/transactionActions.js

```diff
@@ -276,10 +276,12 @@ export function setMaxSpendOnOutput(outputIndex) {
       new BigNumber(0)
     );
 
-    const spendAllAmount = satoshisToBitcoins(
-      new BigNumber(confirmedBalance)
-        .minus(outputs.length > 1 ? totalOutputValue : 0)
-        .minus(estimatedFee)
+    const spendAllAmount = new BigNumber(
+      satoshisToBitcoins(
```

Comment:
I believe this function now expects a string or number which is then converted internally to a BigNumber

Reply:
Fixed in [265c874](https://github.com/unchained-capital/caravan/pull/354/commits/265c874276c226c6d9f57c2792e60bfb47c46018)